flashinfer/fused_moe/cutile/activation.py

```
@@ -0,0 +1,144 @@
+        return relu * relu
+    else:
+        # Unsupported codes are rejected by launch_activation before tracing.
+        return x
+
+
+@ct.kernel
+def _gated_activation(
+    X,
+    OUT,
+    activation_type: ConstInt,
+    I: ConstInt,
```

Ruff (0.16.1)

[error] Ambiguous variable name: `I`

(E741)